tex/LaTeXML/Portfile

```
@@ -17,13 +17,14 @@ long_description \
 
 # Written in Perl, but it is an application, not just modules
 PortGroup           perl5 1.0
-perl5.branches      5.28
+perl5.branches      5.26 5.28 5.30
```

Since it is just an application, not a module intended for others to use, why should it list more than a single perl version?
```suggestion
perl5.branches      5.28
```

---

I would have thought that this would allow the user to run LaTeXML with whichever version of perl they had installed or preferred, rather than force them to install, perl 5.28 in addition to, say, 5.30.

Have I (again) misunderstood the intent of `perl5.branches`?  If so, I'll change it. 

Aside: Is it too early to revbump (is that what it's called?) to perl 5.30?

---

Having more than perl version in `perl5.branches` is only for p5-* perl module ports. For all other ports, you need to choose the current version of perl that is the default for Macports in order to have [repeatable builds](https://trac.macports.org/wiki/ReproducibleBuilds).

---

> Aside: Is it too early to revbump (is that what it's called?) to perl 5.30?

"Revbump" means increasing the `revision` of a port.

I believe we should have a single default version of perl that all port use, to the greatest extent possible. Currently per the perl5 portgroup that's 5.28. [Here](https://trac.macports.org/ticket/58361) is the ticket documenting all that was changed when we switched to 5.28; probably similar steps would be needed when switching to 5.30.